src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -271,7 +272,7 @@ export class SchoolSelectField extends React.Component {
               {`${searchResultsCount} results for ${institutionQuery}`}
             </span>}
             {showSearchResults && showInstitutions && <div>
-              {institutions.map(({ address1, address2, address3, city, country, facilityCode, name, state }, index) => (
+              {institutions.map(({ address1, address2, address3, city, country, facilityCode, name, state, zip }, index) => (
```

Comment:
The BE schema expects "USA" to be "United States". Should we perform that transformation in the reducer on the frontend?

Reply:
Thanks for catching that. How did you catch that, actually? Yeah, I'll see what I can do to make that transform happen on the front end.

Reply:
It looks like sending "USA" instead of "United States" isn't the only potential issue. The search tool returns schools from other countries and those countries come back in ALL CAPS while the backend expects Title Case. Is that an issue we also need to address? Perhaps @jholton could weigh in since this is a question of what the backend can handle.